### DISCUSSION TITLE
fill-current not working for svg file image tag in rails

### DISCUSSION THREAD

**Discussion by joelcahalan** [tailwindlabs/tailwindcss#2533] [Help]
I moved some svgs to seperate files to clean up some html and I am using the `image_tag` helper in rails. The classes specified in the svg file don't register so I applied them to the helper. Sizing classes are working such as h-4 and w-4 but `fill-current text-white` does not change the fill color of the svg. Is there a way to get this to work or do I need to move the svg back to the html files?

**Comment by adamwathan** [comment]
Can't apply classes to SVGs that are loaded through `img` tags unfortunately, only inline SVGs.

**Comment by joelcahalan** [comment]
I found a way around this that is detailed here: https://cobwwweb.com/render-inline-svg-rails-middleman

**Reply by janpeterka** [reply]
This link is sadly now not working

**Reply by wongjn** [reply]
You can find the page in the Internet Archives: https://web.archive.org/web/20201026041448/https://cobwwweb.com/render-inline-svg-rails-middleman
